packages/kit/src/runtime/client/client.js

```diff
@@ -1220,6 +1220,13 @@ export function create_client({ target, session, base, trailing_slash }) {
 				}
 			});
 
+			// fix link hrefs, because browsers will occasionally try to load relative
+			// URLs after a pushState/replaceState, resulting in a 404 — see
+			// https://github.com/sveltejs/kit/issues/3748
```

Comment:
how about linking to the one comment that's actually helpful in a sea of "me too" comments :smile: 
```suggestion
			// https://github.com/sveltejs/kit/issues/3748#issuecomment-1125980897
```